### DISCUSSION TITLE
✨ Add extension to add auto moq and AutoNSubstitute

### DISCUSSION THREAD

**Discussion by ChangeTheCode** [AutoFixture/AutoFixture#1374] [Ideas]
### Description
Expose an Extension method to easily add auto mock without write each time `.Customize(new AutoMoqCustomization())` or to provide this on an common way in the own project. 

### Example
`public static IFixture AddAutoMoq(this IFixture current)`
`public static IFixture AddAutoNSubstitute(this IFixture current)`

Thanks!

**Comment by aivascu** [comment]
@ChangeTheCode thank you for submitting this idea.
Please note that feature requests should be submitted in the [Ideas](https://github.com/AutoFixture/AutoFixture/discussions/categories/ideas) discussions. 

Could you provide some motivation for this feature? As I see it, this would take away the possibility for configuring the customization, and will make it less obvious that there is one.

Moving to discussions.

**Reply by Kralizek** [reply]
Whilst the proposed form hides the customization possibilities, it is still possible to add parameters with default values or an overload accepting a generator function.

Generally I agree that AutoFixture is a bit unfriendly when it comes to feature discovery. i suggested the same improvement in a past PR but it was blocked because some of the maintainers thought that there was no need for it.

**Reply by ChangeTheCode** [reply]
@aivascu  the same like @Kralizek  mentioned here is my point of view. This extension will not prohibit anything of the functionality which is there yet. It will just make the live a bit more comfortable for the autofixture user. 

If this feature is wanted I could provide an PR to contribute this small feature.
